modules/tensor_mechanics/test/tests/rom_stress_update/2drz.i

```diff
@@ -0,0 +1,118 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[AuxVariables]
+  [./temperature]
+    initial_condition = 900.0
+  [../]
+[]
+
+[Modules/TensorMechanics/Master]
+  [./all]
+    strain = FINITE
+    add_variables = true
+    use_automatic_differentiation = true
+  [../]
+[]
+
+[BCs]
+  [./symmy]
+    type = ADPresetBC
+    variable = disp_y
+    boundary = bottom
+    value = 0
+  [../]
+  [./symmx]
+    type = ADPresetBC
+    variable = disp_x
+    boundary = left
+    value = 0
+  [../]
+  [./pressure_x]
+    type = ADPressure
+    variable = disp_x
+    component = 0
+    boundary = right
+    constant = 3.1675e5
+  [../]
+  [./pressure_y]
+    type = ADPressure
+    variable = disp_y
+    component = 1
+    boundary = top
+    constant = 6.336e5
```

Comment:
these BCs were for a thin walled pressure vessel approximation, and we might want to change them to make sure that we get an applied effective stress within the allowable ranges

Reply:
This was modified to produce a stress of 32 MPA, square in the range of applicability.